Geometry_Engine/Compute/PointClustersDBSCAN.cs

```diff
@@ -33,12 +34,14 @@ public static partial class Compute
         /***************************************************/
         /**** public Methods - Vectors                  ****/
         /***************************************************/
-
+        
         public static List<List<Point>> PointClustersDBSCAN(this List<Point> points, double maxDist, int minPointCount = 1)
         {
-            double maxSqrDist = maxDist * maxDist;
-            Func<Point, Point, bool> metricFunction = (x, y) => x.SquareDistance(y) <= maxSqrDist;
-            return points.ClusterDBSCAN(metricFunction, minPointCount);
+            double sqDist = maxDist * maxDist;
+            Func<Point, DomainBox> toDomainBox = a => a.IBounds().DomainBox();
```

Comment:
Can't we simply create a `DomainBox` without calling a chain of methods here?

Reply:
Yes, (but it's a bit ugly)